index.html

```diff
@@ -170,7 +184,7 @@ <h1 class="contact">Contact me</h1>
      project that needs coding.<br> 
     I'd love to help with it!</p>
     <div class="forms">
-    <form method="post" action="">
+    <form method="post" action="https://formspree.io/f/xvojodzg">
         <input type="text" name="name" id="name"  placeholder="Lorem Ipsum" maxlength="30">
```

Comment:
Kindly add the `required` attribute to activate the validation on submission of your form 😄